src/System.Linq.Expressions/tests/ExceptionHandling/ExceptionHandlingExpressions.cs

```diff
@@ -288,93 +282,60 @@ public void NonExceptionDerivedExceptionWrappedInterpreted()
             Assert.Equal("System.Runtime.CompilerServices.RuntimeWrappedException", ex.GetType().FullName);
```

Comment:
It's what the runtime manufactures when a non-Exception-derived object is thrown (e.g. from C++/CLI) and enters into code written in C#/VB.